app/views/breweries/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<h1><div align="center">Breweries Around You</div><h1>
+
+<div id="nav_bar">
+  <%= render partial: "shared/navbar.html.erb" %>
+</div>
+<br>
+
+<% if @user.city == nil or @user.state == nil %>
+  <div id="update_form">
+    <div>We do not have a city/state saved for you, please update</div>
+      <%= form_with url: '/update', method: :patch, local: true do |form| %>
+        <%= form.city :city %>
+        <%= form.label "City" %>
+        <%= form.select_tag :state, options_for_select(us_states) %>        
+        <%= form.label "State" %>
+        <%= form.submit "Submit"%>
+      <% end%>
+  </div>
+<% else %>
+  <% @breweries.each do |brewery| %>
+    <div id="brewery_<%= brewery.id %>">
+      <% if brewery.website == nil %>
+        <li><%= brewery.name %><br>
+      <% else%>
+        <li><%= link_to brewery.name, brewery.website %><br>
```

Comment:
This works great, but we might consider refactoring to add `:target => "_blank"` to open the page in a new window so the user doesn't leave our webpage.